<ui version="4.0" >
 <class>RangeTab</class>
 <widget class="QWidget" name="RangeTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>X Zoom Mode </string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="_xAuto" >
        <property name="text" >
         <string>A&amp;uto</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QRadioButton" name="_xMean" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Mean c&amp;entered - </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Ran&amp;ge:</string>
          </property>
          <property name="buddy" >
           <cstring>_xRange</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_xRange" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3" >
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QRadioButton" name="_xFixed" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>F&amp;ixed -</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Fro&amp;m</string>
          </property>
          <property name="buddy" >
           <cstring>_xMin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_xMin" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>T&amp;o</string>
          </property>
          <property name="buddy" >
           <cstring>_xMax</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_xMax" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="_xSpike" >
        <property name="text" >
         <string>S&amp;pike insensitive auto</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="_xBorder" >
        <property name="text" >
         <string>Auto with bor&amp;der</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Y Zoom Mode </string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QRadioButton" name="_yAuto" >
        <property name="text" >
         <string>&amp;Auto</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="_yBorder" >
        <property name="text" >
         <string>Auto with &amp;border</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="_ySpike" >
        <property name="text" >
         <string>&amp;Spike insensitive auto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QRadioButton" name="_yMean" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Mean &amp;centered - </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Range:</string>
          </property>
          <property name="buddy" >
           <cstring>_yRange</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_yRange" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3" >
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QRadioButton" name="_yFixed" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;Fixed -</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>F&amp;rom</string>
          </property>
          <property name="buddy" >
           <cstring>_yMin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_yMin" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>&amp;To</string>
          </property>
          <property name="buddy" >
           <cstring>_yMax</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_yMax" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_xMean</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>59</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel" >
     <x>186</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_xFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>41</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_xFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>75</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel" >
     <x>227</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_yMean</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>142</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel" >
     <x>181</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_yFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>75</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel" >
     <x>118</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_yFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>79</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel" >
     <x>226</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_xMean</sender>
   <signal>toggled(bool)</signal>
   <receiver>_xRange</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>50</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel" >
     <x>290</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_xFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>_xMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>51</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_xFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>_xMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>66</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel" >
     <x>258</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_yMean</sender>
   <signal>toggled(bool)</signal>
   <receiver>_yRange</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>81</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel" >
     <x>230</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_yFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>_yMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>59</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel" >
     <x>137</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_yFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>_yMax</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>64</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel" >
     <x>267</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
